### PR TITLE
Milestone 3: Add Clear all Completed Task Feature

### PR DESCRIPTION
**Pull request by AyoubM490**
In this Milestone, I:
- Added the Clear All Completed Task Feature
- Used ES6

### CODE REVIEW

File: src/modules/removeCompleted.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const removeCompleted = () => {
+  const labels = document.querySelectorAll('label');
+  labels.forEach((label) => {
+    if (label.classList.contains('checked')) {
+      label.parentNode.parentNode.remove();
+    }
+  });
+};
+
+export default removeCompleted;
```
Comment (Meltrust) [inline src/modules/removeCompleted.js]: According to the project requirements: 

image

Therefore:

- [x] Please, kindly use `Array.filter()` to delete your completed tasks with the "Clear all completed task" button.

That way, we can meet this particular requirement 💪

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import './style.css';
 import addTask from './modules/addTask.js';
 import removeTask from './modules/removeTask.js';
+import removeCompleted from './modules/removeCompleted.js';
+import trashCompleted from './modules/completed.js';
 import { saveStorage, getStorage } from './modules/storage.js';
```
Comment (Meltrust) [inline src/index.js]: - [x] The localStorage functionality is not working as intended.  Let me explain with a video:

https://share.getcloudapp.com/DOum7J4R

The items are not being updated and saved correctly in `localStorage`, upon deletion of some of them.
Please, check your `localStorage `related algorithms to achieve the correct functionality.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @AyoubMs,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice readme ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (AyoubM490) [reply src/index.js]: Done

File: src/modules/removeCompleted.js (hunk 1 above)
Comment (AyoubM490) [reply src/modules/removeCompleted.js]: Done

File: src/index.js [hunk 3]
```diff
@@ -1,86 +1,104 @@
+/* eslint-disable radix */
+/* eslint-disable import/no-cycle */
+
```
Comment (Cathella) [inline src/index.js]: Linters help us to write clean code so disabling them means we are doing ourselves a disservice. 

- [x] Please kindly enable them and fix them instead.

File: src/modules/addlist.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { saveStorage, getStorage } from './storage.js';
+// eslint-disable-next-line import/no-cycle
```
Comment (Cathella) [inline src/modules/addlist.js]: Linters help us to write clean code so disabling them means we are doing ourselves a disservice. 

- [x] Please kindly enable them and fix them instead.

File: src/modules/completed.js [hunk 5]
```diff
@@ -1,9 +1,12 @@
+/* eslint-disable import/no-cycle */
```
Comment (Cathella) [inline src/modules/completed.js]: Linters help us to write clean code so disabling them means we are doing ourselves a disservice. 

- [x] Please kindly enable them and fix them instead.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @AyoubMs 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (AyoubM490) [reply src/index.js]: Done

File: src/modules/addlist.js (hunk 4 above)
Comment (AyoubM490) [reply src/modules/addlist.js]: Done

File: src/modules/completed.js (hunk 5 above)
Comment (AyoubM490) [reply src/modules/completed.js]: Done

**Review by ZeenatLawal — APPROVED** [review]
Hi @AyoubMs, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**